modules/tools/cluster-autoscaler/charts/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+appVersion: 1.18.1
+description: Scales Kubernetes worker nodes within autoscaling groups.
+engine: gotpl
+home: https://github.com/kubernetes/autoscaler
+icon: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png
+maintainers:
+  - email: e.bailey@sportradar.com
+    name: yurrriq
+  - email: mgoodness@gmail.com
+    name: mgoodness
+  - email: guyjtempleton@googlemail.com
+    name: gjtempleton
+  - email: scott.crooks@gmail.com
+    name: sc250024
+name: cluster-autoscaler
+sources:
+  - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
+type: application
+version: 9.4.0
```

Comment:
do we need to vendor this chart?  looks like its available here: https://artifacthub.io/packages/helm/cluster-autoscaler/cluster-autoscaler-chart

Reply:
The artifactory hub is a copy from the deprecated repo and I'm not sure it is being actively maintained. The cluster-autoscaler repo links to the [official chart](https://github.com/kubernetes/autoscaler/tree/master/charts) in the repo which is more up to date.

Reply:
Took another look and it is being maintained but there are two different charts https://artifacthub.io/packages/helm/cluster-autoscaler/cluster-autoscaler. 

Updated module to use published chart.